packages/flutter/lib/src/material/text_selection_toolbar_text_button.dart

```diff
@@ -81,21 +81,24 @@ class TextSelectionToolbarTextButton extends StatelessWidget {
   ///
   /// Standard Material [TextSelectionToolbar]s have buttons with different
   /// padding depending on their position in the toolbar.
-  static EdgeInsets getPadding(int index, int total) {
+  static EdgeInsetsGeometry getPadding(int index, int total) {
```

Comment:
I've started the Google tests on this PR just to make sure this doesn't break anyone.